north/src/runtime/mod.rs

```diff
@@ -113,6 +113,22 @@ pub async fn run(config: &Config) -> Result<()> {
         state.applications.len()
     );
 
+    // handle SIGINT and SIGTERM
+    let tx_interrupt = tx.clone();
+    ctrlc::set_handler(move || {
```

Comment:
Installing a signal handler catches a signal. Think this means that the signal is not sent to childs? This is something we would need.